### ISSUE TITLE
Accessibility: Video block placeholder image missing Alt

### ISSUE THREAD

**Issue by rnunez80** [plone/volto#4747]
Accessibility Issues
WCAG 2.1 Level A
Alternative (alt) text is read to users utilizing screen readers. Alternative text should be provided to describe what is present in the image so that users who are unable to see have an equal experience as sighted users. Quick tip: If the image is being utilized as a link with no other text, the alternative text should be the location of the link. If the image is of words, the alternative text should be the words contained in the image.
Impact: critical
Violations
WCAG 2.1 Level A: [1.1.1 Non-text Content](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html)
Section 508: 1194.22 (a) Text equivalent for non-text elements
Element Location
.computer.mobile.tablet:nth-child(1) > div > .video.block.align > .video-inner > .\31 6\:9.embed.ui > .placeholder

Element Source
<img class="placeholder" src="https://img.youtube.com/vi/videoID/sddefault.jpg">

Failure Summary
Fix any of the following: Image does not have an alt attribute. To markup the image as decorative, override the semantics by adding role="none" or role="presentation"

**Comment by maniksharma424** [comment]
hey rnune80 i would like to fix this issue

**Comment by stevepiercy** [comment]
@maniksharma424 
please read and follow [Contributing to Plone](https://6.docs.plone.org/contributing/index.html) and [First-time contributors](https://6.docs.plone.org/contributing/first-time.html).
